charts/policy-reporter/templates/deployment.yaml

```diff
@@ -58,10 +58,12 @@ spec:
               containerPort: 8080
               protocol: TCP
           livenessProbe:
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
             httpGet:
```

Comment:
Ich think you can remove this as it is already defined in the values.yaml as default